config/environments/test.rb

```diff
@@ -118,6 +118,10 @@
   PDF_STORAGE = "#{::Rails.root.to_s}/data/test/pdfs"
 
   ###################################################################
+  # Location of the public and private key for the git user on the system
+  GITOLITE_SETTINGS = { public_key: "/home/git/git.pub",
```

Comment:
Leaving this as double quoted strings to be consistent with the rest of the file. Hound complains either-way https://github.com/MarkUsProject/Markus/pull/2121#discussion-diff-28031347